20/Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM buildpack-deps:stretch
+nFROM buildpack-deps:stretch
 
-ENV OTP_VERSION="20.3.8.16"
+ENV OTP_VERSION="20.3.8.17" \
+    REBAR_VERSION="2.6.4" \
+    REBAR3_VERSION="3.8.0"
```

Comment:
as I felt from a previous comment this is like an over-drinking kid: normally I would only concatenate the consecutive `ENV` lines, because the line order of defining `ENV` still matters to later `RUN` build lines:

```Dockerfile
ENV env1=var1
RUN building something which is relying on env1 only, but might break if running with unexpected env2/...
ENV env2=var2
ENV env3=var3
RUN building something else
```

another advanced Docker images usage is I can show `docker image history ...` to show the layers of an image, and run something from a middle layer of an image:

```console
$ docker image history ...
<sha1>  ...
<sha2> ...
<sha3> ...
```

I can run from the middle like `docker container run sha2 ...`  and expect only those `ENV` set before the lines,

Please do not move ENV cross RUN commands, that is not correct